timescaledb/how-to-guides/skipscan.md

```diff
@@ -0,0 +1,84 @@
+# SkipScan
+SkipScan improves query times for `DISTINCT` queries. It works on PostgreSQL
+tables, TimescaleDB hypertables, and TimescaleDB distributed hypertables.
+SkipScan is included in TimescaleDB 2.2.1 and above.
+
+## Faster DISTINCT queries using SkipScan
+When you query your database to find the most recent value of an item, you
+usually use a `DISTINCT` query. For example, you might want to find the latest
+stock or cryptocurrency price for each of your investments. Or you might have graphs
+and alarms that repeatedly query the most recent values for every device or
+service.
```

Comment:
Hmm i dont think this is actually true. I think you would almost never use a DISTINCT query here because the performance of that query would be really bad without skipscan optimization.

Reply:
```suggestion
To query your database and find the most recent value of an item, you
could use a `DISTINCT` query. For example, you might want to find the latest
stock or cryptocurrency price for each of your investments. Or you might have graphs
and alarms that repeatedly query the most recent values for every device or
service.
```